gwak/train/dataloader.py

```diff
@@ -604,7 +605,7 @@ def make_dataset(self, fnames, coincident, mode):
             mode=mode,
             glitch_root=self.glitch_root,
             ifos=self.ifos,
-            remake_cache=self.remake_cache
```

Comment:
i think we need to pass here `remake_cache`, its a new feature that @SamBT implemented to force re-running of glitch finding during data generation in case we are training with a different kernel size. Could you pls put it back